include/Enemy.h

```diff
@@ -39,7 +40,7 @@ class Enemy : public Entity {
     int Action();
     void DisplayHUD();
 
-    std::vector<int> GetDrops();
+    std::map<ITEMTYPE, int> GetDrops();
```

Comment:
Any reason why not `std::unordered_map`? Technically it's better with an average O(1) complexity for insertion/deletion (though it doesn't matter that much with a collection this small).

Reply:
@tagniam yes you are right, I have redefined my solution with unordered_map